api/query/README.md

```diff
@@ -8,7 +8,86 @@ As outlined below, the `/api/search` endpoint takes a structured query object. w
 UI contains a search-box that converts a search syntax into the required structured query.
 Listed below are the "atoms" that can be used in a search.
 
-### Status
+### Root queries
+
+By default, a search query will be implicitly treated as an `exists` query (a disjunction
+across each of the runs separately). However, there are several other root query types that
+can be invoked by wrapping the query(s), including explicitly wrapping with `exists`.
+
+If multiple root queries are used, they are combined with `AND`, i.e. each query must
+separately be true. e.g.
+
+    count>1(status:!pass) none(status:missing)
+
+Requires that more than one non-pass result is present, and none of the results is missing.
+
+#### Exists
+
+As stated above, exists is the implicit default.
+
+    exists([query1] [query2])
+
+Combines the filters such that there exists some result in the row that satisfies each query.
+
+#### All and None
+
+> BETA: This feature is under development and may change without warning.
+
+    all([query1] [query2])
+
+Combines filters such that they must all apply to all runs.
+
+    none([query1] [query2])
+
+Combines filters such that they must not _all_ apply to _any_ single run.
+
+#### Sequential
+
+    seq([query1] [query2] [...])
+
+Combines filters such that they must apply to runs sequentially. This is mainly
+useful when there are multiple runs with the same product, e.g. to find a regression
+
+    seq(status:pass status:fail)
+
+#### Count
+
+> BETA: This feature is under development and may change without warning.
+
+    count:[number]([query1] [query2])
+
+Requires that the number of results matching the given query/queries is precisely
+the given count. For example, this search atom can be used to find cases where
+exactly one result is a failure:
+
+    count:1(status:fail)
```

Comment:
Do you want to document `count=` here as well?

Reply:
No, I think it's fine; the colon version is documented for persons reading the docs, and the equals is supported in case people infer equality from the inequality options.